src/knex-processor.ts

```diff
@@ -0,0 +1,54 @@
+import * as Knex from "knex";
+
+import OperationProcessor from "./operation-processor";
+import { Resource } from "./types";
+
+export default class KnexProcessor<
+  ResourceT extends Resource
+> extends OperationProcessor<ResourceT> {
+  private knex: any;
```

Comment:
@Exelord can we be explicit about the type for this property? It should be:

```suggestion
  private knex: Knex;
```